Source/gravity/Gravity.cpp

```diff
@@ -3574,9 +3574,9 @@ Gravity::update_max_rhs()
             coeffs[lev].resize(AMREX_SPACEDIM);
 
             for (int i = 0; i < AMREX_SPACEDIM ; i++) {
-                coeffs[lev][i].reset(new MultiFab(amrex::convert(grids[lev],
-                                                  IntVect::TheDimensionVector(i)),
-                                                  dmap[lev], 1, 0));
+                coeffs[lev][i] = std::make_unique<MultiFab>(amrex::convert(grids[lev],
+                                                                           IntVect::TheDimensionVector(i)),
```

Comment:
Too much indentation

Reply:
I think it's correct, since that is an argument of `amrex::convert()`

Reply:
Oh, my mistake.